src/js/video.js

```diff
@@ -386,90 +418,32 @@ Object.defineProperty(videojs.middleware, 'TERMINATOR', {
 });
 
 /**
- * A suite of browser and device tests from {@link browser}.
+ * A reference to the {@link module:browser|browser utility module} as an object.
  *
  * @type {Object}
- * @private
+ * @see  {@link module:browser|browser}
  */
 videojs.browser = browser;
 
 /**
- * Whether or not the browser supports touch events. Included for backward
- * compatibility with 4.x, but deprecated. Use `videojs.browser.TOUCH_ENABLED`
- * instead going forward.
+ * Use {@link module:browser.TOUCH_ENABLED|browser.TOUCH_ENABLED} instead; only
+ * included for backward-compatibility with 4.x.
  *
- * @deprecated since version 5.0
+ * @deprecated Since version 5.0, use {@link module:browser.TOUCH_ENABLED|browser.TOUCH_ENABLED instead.
  * @type {boolean}
  */
 videojs.TOUCH_ENABLED = browser.TOUCH_ENABLED;
 
-/**
- * Subclass an existing class
```

Comment:
I think this description should probably be added into the extend file, as the extend file has a very technical description.